frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -510,6 +498,35 @@ The interval value is in seconds and can range from 3 to 20 seconds.]]),
             end,
         })
     end
+    table.insert(menu_items.long_press.sub_item_table, {
+        text_func = function()
+            return T(_("Highlight very long-press interval: %1 s"),
+                G_reader_settings:readSetting("highlight_long_hold_threshold_s", 3))
+        end,
+        keep_menu_open = true,
+        callback = function(touchmenu_instance)
+            local SpinWidget = require("ui/widget/spinwidget")
+            local items = SpinWidget:new{
+                title_text = _("Highlight very long-press interval"),
+                info_text = _("If a touch is not released in this interval, it is considered a very long-press. On document text, single word selection will not be triggered."),
```

Comment:
Should `very long-press` have another hyphen ? `very-long-press` ? or is this just silly ?

Also, for the info_text, may be
_If a ~~touch~~ long-press is not released_ ... ?

Reply:
I'd overlooked that. Indeed, I'd opt for either very-long-press or very long press.